python/lsst/dbb/buffmngrs/handoff/defaults.py

```diff
@@ -0,0 +1,50 @@
+# This file is part of dbb_buffmngrs_handoff.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from dataclasses import dataclass
+
+
+__all__ = ["Defaults"]
+
+
+@dataclass
+class Defaults:
+    """Default values for general settings.
+    """
+
+    chunk_size = 10
+    """Number of files to processes by a transfer thread in a single session.
```

Comment:
I'd say that for the new version the most accurate description of the `chunk_size` parameter would be
> a number of messages/items a manager's command retrieves from the input message queue in each iteration of its main loop (when applicable [1])
It just seemed "insanely" specific for a *general* option description.

I think I'll go with
> """Number of messages to retrieve from the input message queue.
>
> In the context of file transfers, it corresponds to maximal number of files that will be transfer to the endpoint site with a single transfer command.
>"""

[1] Not every manager's command has an input message queue.

Reply:
This comment is much better.  But still I would like to enforce that it doesn't wait until it can grab chunk_size messages. 
 Maximum number of messages/items to retrieve from the input message queue?